### PR TITLE
XCM Refactor

### PR DESCRIPTION
**Pull request by CrackTheCode016**
Since V3 is out, we need to not opinionate the content to "v3 is coming out".

### CODE REVIEW

**Comment by CrackTheCode016** [comment]
@filippoweb3 @DrW3RK 

I am thinking of removing the XCMP section here and creating it in "Future Implementations" as it seems more appropriate. It also makes the page more concise, while highlighting relevant, current information about XCM.

Wydt?

**Comment by DrW3RK** [comment]
@CrackTheCode016 Yes. Please include a link to XCMP section in this doc and move the section to future implementation.

**Comment by CrackTheCode016** [comment]
@DrW3RK Just saw your comment - I'll do this in a separate PR, so as to not break anything here while it transitions

File: docs/learn/learn-xcm-instructions.md [hunk 1]
```diff
@@ -7,9 +7,19 @@ keywords: [cross-consensus, XCM, XCMP, interoperability, communication]
...
+import Tabs from "@theme/Tabs";
+
 This page can also be viewed at the [`xcm-format`](https://github.com/paritytech/xcm-format)
 repository, where each instruction and register is explained in-depth.
 
+:::caution
+
+For a more practical approach to utilizing XCM registers and instructions, refer to the
+[XCM Docs](https://paritytech.github.io/xcm-docs/). Please keep in mind that these are currently
+under development.
```
Comment (profradha) [inline docs/learn/learn-xcm-instructions.md]:
```suggestion
[XCM Docs](https://paritytech.github.io/xcm-docs/). Please keep in mind that XCM is
under active development.
```

File: docs/learn/learn-xcm-instructions.md [hunk 2]
```diff
@@ -66,3 +78,52 @@ repository, where each instruction and register is explained in-depth.
 - `UnlockAsset`
 - `NoteUnlockable`
 - `RequestUnlock`
+
+## Instructions Application Example
+
+The following presents the practical mapping of instructions to some of the core functionality to
+XCM [[sdf](./learn-xcm.md/#core-functionality-of-xcm)](./learn-xcm.md/#core-functionality-of-xcm).
```
Comment (profradha) [inline docs/learn/learn-xcm-instructions.md]:
```suggestion
XCM [sdf](./learn-xcm.md/#core-functionality-of-xcm).
```

**Review by DrW3RK — APPROVED** [review]
Thanks for updating the XCM doc!

**Pull request merged**